plugins/kubernetes_ng/app/javascript/widgets/app/routes/clusters/-components/ClusterWizard/WorkerGroupSection.tsx

```diff
@@ -222,6 +227,7 @@ const WorkerGroupSection = ({ workerGroup, index, totalWorkers, onChange, onDele
                   })
                 }
                 onBlur={() => validateSingleField(`workers.${workerGroup.id}.machineImage.version`)}
```

Comment:
The validation shouldn't happen and no error message should be shown while the select is disabled. In your first screenshot you have the situation that this select is disabled but it still shows an error.